cpp/src/IceDB/IceDB.h

```diff
@@ -82,17 +82,17 @@ namespace IceDB
     // Read [out] T from [in] MDB_val using [in] context C
     // static void read(T& , const MDB_val&, const C&);
     //
-    // Write [in] T into [out] MDB_val using [in] context C
+    // Write [in] T into [out] MDB_val
```

Comment:
We don't need all this flexibility since IceDB is an internal layer used only by a few services.